docs/run_test/troubleshoot_failures.rst

```diff
@@ -0,0 +1,260 @@
+the end of a successful test run.
+
+  -  **FAILED**
+  
+     FAILED tests are tests that did not finish successfully and
+     terminated because of failures like ``LISA exceptions`` or
+     ``Assertion failure``. You can use them to trace where the problem
+     was and why the problem happened.
+
+  -  **PASSED**
+
+     PASSED tests are tests that passed, or at least partially passed,
+     with a special ``PASSException`` that warns there are minor errors in
+     the run but they do not affect the test result.
+
+  -  **SKIPPED**
+
+     SKIPPED tests are tests that did not start and would no longer run.
+     They suggest failure to meet some requirements in the environments
+     involved with the test.
+
+  -  **ATTEMPTED**
+
+     ATTEMPTED tests are a special category of FAILED tests because of
+     known issues, which are not likely to be fixed soon.
+
+- **Intermediate results**
+
+An intermediate result shows information of an unfinished test. It will
+show up when a test changes its state. If a test run terminates because
+of error or exception prior to running a test case, only the
+intermediate result will be provided.
+
+  -  **QUEUED**
+
+     QUEUED tests are tests that are created, and planned to run (but have
+     not started yet). They are pre-selected by extension/runbook
+     criteria. You can check log to see which test cases are included by
+     such criteria. They suggest that there are some tests waiting to be
+     performed.
+
+     QUEUED tests will try to match every created environment. They will
+     move forward to ASSIGNED if they match any, and to SKIPPED if they
+     match none of the environments.
+
+  -  **ASSIGNED**
+  
+     ASSIGNED tests are tests that are assigned to an environment, and
+     will start to run, if applicable, once the environment is
+     deployed/initialized. They suggest some environmental setting up is
+     going on.
+
+     ASSIGNED tests will end with FAILED if the environment fails to
+     deploy. Otherwise, they move forward to RUNNING. They will also move
+     backward to QUEUED if the environment is deployed and initialized
+     successfully.
+
+  -  **RUNNING**
+  
+     RUNNING tests are tests that are in test procedure.
+     RUNNING tests will end with one of the following final results.
+
+Log Files
+--------------------
+
+After a test run, the LISA log file will be generated. The log file can
+be found in the `runtime/log` directory that is generated after test
+runs.  Navigate sub-folders until you find the log with a timestamp
+corresponding to the time of the test run.  Inside the log's timestamped
+folder, the contents are further split by environment and test case. 
+If the test run only has a few cases, the full log (`lisa-<timestamp>.log`)
+may be easier to read. If it is run with concurrency, the split logs may
+be easier to read.  The logs will show INFO and above levels by default.
```

Comment:
>> The logs will show INFO and above levels by default.
 The log is always show with DEBUG level, it's not adjustable. We can just remove this line. It's not a useful information.